lib/fluent/plugin/out_influxdb.rb

```diff
@@ -55,14 +55,24 @@ def configure(conf)
                                               time_precision: @time_precision,
                                               use_ssl: @use_ssl,
                                               verify_ssl: @verify_ssl
-    
+
+    $log.info "Connecting to database: #{@dbname}, host: #{@host}, port: #{@port}, username: #{@user}, password = #{@password}, use_ssl = #{@use_ssl}, verify_ssl = #{@verify_ssl}"
+
+    @influxdb = InfluxDB::Client.new @dbname, host: @host,
+                                     port: @port,
+                                     username: @user,
+                                     password: @password,
+                                     async: false,
+                                     time_precision: @time_precision,
+                                     use_ssl: @use_ssl,
+                                     verify_ssl: @verify_ssl
+
     existing_databases = @influxdb.list_databases.map { |x| x['name'] }
```

Comment:
If something is incorrect in connection config (host, port, ssl, verify_ssl), this line will print errors to stdout every several seconds (because of this: https://github.com/influxdata/influxdb-ruby/blob/7b38df9f0fabb8d0cc4f5eae09f33a10f77ad6a1/lib/influxdb/client/http.rb#L58). Best solution - split connection for verification step (retry == 1) and `production` connection (retry == -1). But I'm not familiar with Ruby at all to do it properly.

I believe that my PR makes plugin better, because now error is very visible (extremely visible :) ). But, of course, it would be much better if somebody can do verification in best way.